.changeset/nervous-plants-refuse.md

```diff
@@ -0,0 +1,6 @@
+---
+'myst-cli': minor
+'myst-spec-ext': minor
```

Comment:
Thank you! I think that we can have a `patch` change for these as they are backwards compatible additions?

```suggestion
'myst-cli': patch
'myst-spec-ext': patch
```

Reply:
That's true. I should get myself more familiar with changeset tools. :eyes:

Reply:
All good! It is super helpful in these monorepos, it builds the changelog as well as bumps all of the dependencies between packages. That and turbo repo are pretty big gamechangers for mono-repo development. :)